shm/Player.hpp

```diff
@@ -3,17 +3,25 @@
 #include <memory>
 
 #include "Cargo.hpp"
+#include "Delegate.hpp"
 #include "Ship.hpp"
 
-class Player {
+class Player : public Delegate {
 public:
-    Player(std::unique_ptr<Ship> ship, size_t money);
+    Player(std::unique_ptr<Ship>& ship, size_t money);  //I need to pass ship as & to construct Player in main() like this:
```

Comment:
Is that comment necessary for final version?

Reply:
I just wanted to let you know what's going on here and why this line has changed. But actually I could do it here on GH. I'll remove it.